packages/grpc-js/src/make-client.ts

```diff
@@ -155,7 +164,7 @@ export function makeClientConstructor(
     ServiceClientImpl.prototype[name] = methodFunc;
     // Associate all provided attributes with the method
     Object.assign(ServiceClientImpl.prototype[name], attrs);
-    if (attrs.originalName && attrs.originalName !== '__proto__') {
+    if (attrs.originalName && isPrototypePolluted(attrs.originalName)) {
```

Comment:
2' condition sentence was `attrs.originalName !== '__proto__'` , so the correct version should be `!isPrototypePolluted(attrs.originalName)`

Reply:
Thank you @Mik317, I've fixed the mistake. Appreciate the support.

Cheers